### PR TITLE
Update OMSDK demo description

### PR DESCRIPTION
**Pull request by mimcz**
Getting started text needs to be updated as the library is now bundled.

### CODE REVIEW

File: player/ad-event-verification-omsdk/index.html [hunk 1]
```diff
@@ -70,19 +70,10 @@ <h2>
...
-            1. Acquire the OM SDK files from the
-            <a href="https://tools.iabtechlab.com/omsdk">
-                IAB Tech Lab Tools Portal
-            </a>
-        </p>
         <p class="mb-3">
-            2. Include the following scripts in your page
+            1. Include the following scripts in your page
         </p>
         <ul class="ml-4">
```
Comment (jmsn) [inline player/ad-event-verification-omsdk/index.html:76]: I think we should get rid of this `<ul>` here as it now has only a single list entry and instead consolidate with line 74 🙂

**Review by jmsn — COMMENTED** [review]
Nice!

Can we please make these additional changes:
- As discussed internally the `Partner Version` is actually the version of the app that integrates the OMSDK and not the OMSDK version. We should change our default value of `1.3.25` to something more generic (e.g. `1.0.0`) to avoid confusion.

- The link here https://github.com/bitmovin/demos/blob/6a6e4d9e2375057d6d0f5174a89ccfd77ef5ae6a/player/ad-event-verification-omsdk/index.html#L95-L97 should actually point to https://bitmovin.com/docs/player/api-reference/web/web-sdk-api-reference-v8#/player/web/8/docs/interfaces/advertising.omsdktracker.html which defines the OMSDK tracker interface. Could you please change it? 🙂

**Comment by mimcz** [comment]
> Nice!
> 
> Can we please make these additional changes:
> 
> * As discussed internally the `Partner Version` is actually the version of the app that integrates the OMSDK and not the OMSDK version. We should change our default value of `1.3.25` in the demo to something more generic (e.g. `1.0.0`) to avoid confusion. Here https://github.com/bitmovin/demos/blob/6a6e4d9e2375057d6d0f5174a89ccfd77ef5ae6a/player/ad-event-verification-omsdk/index.html#L36
>    and here https://github.com/bitmovin/demos/blob/6a6e4d9e2375057d6d0f5174a89ccfd77ef5ae6a/player/ad-event-verification-omsdk/page.js#L23
> * The link here https://github.com/bitmovin/demos/blob/6a6e4d9e2375057d6d0f5174a89ccfd77ef5ae6a/player/ad-event-verification-omsdk/index.html#L95-L97
>    should actually point to https://bitmovin.com/docs/player/api-reference/web/web-sdk-api-reference-v8#/player/web/8/docs/interfaces/advertising.omsdktracker.html which defines the OMSDK tracker interface. Could you please change it? 🙂

Thx @jmsn makes sense. I updated it according to your suggestions.

**Review by jmsn — APPROVED** [review]
LGTM!

**Pull request merged**